src/ngx-slider/lib/slider.component.ts

```diff
@@ -861,7 +952,7 @@ export class SliderComponent implements OnInit, AfterViewInit, OnChanges, OnDest
     this.updateDisabledState();
     this.updateAriaLabel();
     this.calculateViewDimensions();
-    this.refocusPointerIfNeeded();
+    // this.refocusPointerIfNeeded();
```

Comment:
@danisss9 Can you follow up on this one? If it really solves a problem, then it should be removed, not commented out.

Reply:
I don't remember what the issue was, seems to have been from this commit: [prevent infinite loop detectChanges -> ngOnChanges -> detectChanges](https://github.com/danisss9/ngx-slider/commit/c096d47988e63cf3618075a2cfb443ffd4b1b8a5)

We should try uncomment and retest it.

Reply:
Also resolving this one. I have a rewrite of this function on ng17 branch.